File: tests/fixtures/javascript/comments.js
// FIXTURE: JavaScript file with various comment types
// TESTS: Minimal mode comment stripping

// This is a regular single-line comment (STRIP)
/* This is a regular block comment (STRIP) */

/**
 * This is a JSDoc comment (KEEP)
 * @param {number} x - first number
 * @param {number} y - second number
 * @returns {number} the sum
 */
function add(x, y) {
    // This comment is inside a function body (KEEP)
    const result = x + y; // inline comment in body (KEEP)
    return result;
}

// Another regular comment (STRIP)

/**
 * A documented class (KEEP)
 */
class Calculator {
    /** Constructor doc (KEEP) */
    constructor(value) {
        this.value = value;
    }

    /**
     * Add method doc (KEEP)
     */
    add(x) {
        // body comment (KEEP)
        return this.value + x;
    }
}

/* Regular block comment at module level (STRIP) */




// Test blank line normalization: 4+ blank lines above (normalize to 2)

const VERSION = "1.0.0";

File: tests/fixtures/csharp/comments.cs
// FIXTURE: C# file with various comment types
// TESTS: Minimal mode comment stripping

using System;

// This is a standalone comment (STRIP)

/* Block comment at module level (STRIP) */

/// <summary>
/// XML doc comment for Add method (KEEP)
/// </summary>
/// <param name="a">First number</param>
/// <param name="b">Second number</param>
/// <returns>Sum of a and b</returns>
public static int Add(int a, int b)
{
    // This comment is inside a method body (KEEP)
    int result = a + b; // inline comment in body (KEEP)
    return result;
}

// Regular comment between members (STRIP)

/** Block doc comment for Calculator class (KEEP) */
public class Calculator
{
    private int _value;

    /// <summary>Constructor doc (KEEP)</summary>
    public Calculator(int value)
    {
        _value = value;
    }

    /// <summary>Add method doc (KEEP)</summary>
    public int Add(int x)
    {
        // body comment (KEEP)
        return _value + x;
    }
}

/* Standalone block comment at namespace level (STRIP) */




// Test blank line normalization: 4+ blank lines above (normalize to 2)

public const int VERSION = 1;
